scripts/covid_tracking_project/historic_us_data/preprocess_csv.py

```diff
@@ -41,18 +41,15 @@
                 ('CumulativeCount_MedicalConditionIncident'
                  '_COVID_19_PatientRecovered'):
                     row_dict['recovered'],
-                ('CumulativeCount_MedicalConditionIncident',
-                 '_COVID_19_PatientDeceased'):
+                ('CumulativeCount_MedicalConditionIncident', '_COVID_19_PatientDeceased'):
```

Comment:
If these fit on a single line, we might as well concatenate them.  Here and below.

Reply:
This is how the formatter spit things out.